num_enum_derive/Cargo.toml

```diff
@@ -26,7 +26,7 @@ default = ["std"]  # disable to use in a `no_std` environment
 features = ["external_doc"]
 
 [dependencies]
-proc-macro2 = "0.4.30"
+proc-macro2 = "1"
 proc-macro-crate = "0.1.4"
-proc-quote = "0.2.2"
-syn = "0.15.44"
+proc-quote = "0.3"
```

Comment:
Actually, now that `syn/quote/proc-macro2` have gone stable, I don't know if there is a reason to keep using `proc-quote`: `quote 1.0` features the few advantages that `proc-quote` had, I think. So it may be worth exploring reverting back to `quote`.

Reply:
Done!